Source/Core/Core/PowerPC/JitArm64/JitArm64_SystemRegisters.cpp

```diff
@@ -218,7 +218,7 @@ void JitArm64::twx(UGeckoInstruction inst)
   fpr.Flush(FlushMode::MaintainState);
 
   LDR(IndexType::Unsigned, WA, PPC_REG, PPCSTATE_OFF(Exceptions));
-  ORR(WA, WA, 24, 0);  // Same as WA | EXCEPTION_PROGRAM
+  ORRI2R(WA, WA, EXCEPTION_PROGRAM);
```

Comment:
Worth noting that there was a bug in the previous code, which actually OR'd with EXCEPTION_PERFORMANCE_MONITOR (0x100), not EXCEPTION_PROGRAM (0x80).